### ISSUE TITLE
Let MI remember window size and position on screen

### ISSUE THREAD

**Issue by pstein** [lordmulder/mediainfo-gui#7]
After MI is started I resize the window and move it to another position on screen.

Unfortunately MI does not remember the new size and pos.

Can this remembering added in the next release?

**Comment by lordmulder** [comment]
It would be possible to add this, as an _option_. But, at this time, MediaInfo GUI does **not** even create/read a configuration file where such options (or the previous window position) could be persisted. So _maybe_ in some future version...

**Comment by githottub** [comment]
Since probably most users will maximize the window (which is usually necessary to see as much of the media info as possible) a good compromise would be to default the app to maximized. This could be coded without any need to create/read a configuration file.

**Comment by lordmulder** [comment]
Simply create a _shortcut_ to `MediaInfoXP.exe`, edit the properties of the shortcut and change _"Run:"_ to "_Maximized"_ 😏

**Comment by githottub** [comment]
Yeah, I know but there will be many others who aren't aware of this method. Therefore, I was thinking if you just coded it to open maximized, since that's how most people will need to use it anyway, it would save you from dealing with future issues similar to this one being posted. Just a thought...

**Comment by lordmulder** [comment]
This is kind of a "standard" method :smirk:

There is a reason why the **`main()`** function for Windows GUI applications is defined as:
```
int WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nShowCmd);
```

Note the **`nShowCmd`** parameter that is passed to the applications `main()` function from the system!

https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-winmain
